workspaces/post-leetcode-potd-to-discord/src/getUnixTimestamp.ts

```diff
@@ -0,0 +1,5 @@
+type DateType = "t" | "T" | "d" | "D" | "f" | "F" | "R";
+
+export function getUnixTimestamp(timestamp: number, type?: DateType): string {
```

Comment:
It might be nice to somehow make it clear from the function name that this is Discord-specific. Perhaps `formatTimestampForDiscord`? ("Unix" is usually clear from "timestamp")

Reply:
ack!